app/main.py

```diff
@@ -2,0 +1,8 @@
+def copy_file(command: str) -> None:
+    text = command.split(" ")
```

Comment:
The task checklist suggests using a consistent style of quotes, preferably double quotes. Ensure that all string literals use double quotes for consistency.